install_files/ansible-base/roles/validate/tasks/validate_ossec_info.yml

```diff
@@ -0,0 +1,69 @@
+---
+- name: Validate OSSEC Admin email address.
+  assert:
+    that:
+      - "'@' in ossec_alert_email"
+      - ossec_alert_email != "ossec@ossec.test"
+    msg: >-
+      OSSEC Admin email address must be valid.
+      This address is used to receive OSSEC notifications.
+
+- name: Validate SASL username for OSSEC config.
+  assert:
+    that:
+      # Ensure not the dummy value specified in example vars file.
+      - sasl_username != "test"
+      - "'@' not in sasl_username"
+
+- name: Validate SASL password for OSSEC config.
+  assert:
+    that:
+      # Ensure not the dummy value specified in example vars file.
+      - sasl_password != "password123"
+      - sasl_password|length >= 8
+    msg: >-
+      Password for OSSEC email account must be strong.
```

Comment:
I dunno if this is a useful validation test. I mean, its a good security-practice check but what if the specific account an org is using has 8 characters and there is ? That kind of validation seems better left up to the organization and their email provider rather than during the setup process. The message is getting encrypted so is it our place to mandate the length of the SMTP account to send the email out?